app/ForkedStar.tsx

```diff
@@ -0,0 +1,58 @@
+import { env } from "@/env.mjs";
+import { GoRepoForked } from "react-icons/go";
+import { GoStar } from "react-icons/go";
+
+export default async function ForkedStar() {
+  let forkCount = 0;
+  let starCount = 0;
+
+  const data = await getData();
+
+  data?.map((ls: any) => {
```

Comment:
Avoid usage of `any` types. This won't be needed as octokit responses are typesafe.